ServerCode/application/AppServer.py

```diff
@@ -161,31 +155,40 @@ def file_upload(self):
         else:
             return jsonify({"status": "error", "message": "Only PDF files are allowed"})
 
-    def process_pdf(self, pdf_file, file_id):
+
+    def process_pdf(self, pdf_file, file_id, sheet_type):
```

Comment:
nice job implementing how the pdf will be processed depending on the custom or regular scantron